src/main/java/de/hysky/skyblocker/utils/container/ContainerSolverManager.java

```diff
@@ -48,7 +49,8 @@ public class ContainerSolverManager {
 			new NewYearCakeBagHelper(),
 			NewYearCakesHelper.INSTANCE,
 			ChocolateFactorySolver.INSTANCE,
-			new ReorderHelper()
+			new ReorderHelper(),
+      			new BitsHelper()
```

Comment:
I would use `BitsHelper.INSTANCE` like the in the `TooltipManager` class

Reply:
Ima go trough those and mark as resolved as I change them in IDE

changed that